wagtail/admin/tests/test_edit_handlers.py

```diff
@@ -1522,6 +1523,42 @@ def test_invalid_inlinepanel_declaration(self):
             )
 
 
+class TestNonOrderableInlinePanel(WagtailTestUtils, TestCase):
+    fixtures = ["test.json"]
+
+    def setUp(self):
+        self.request = RequestFactory().get("/")
```

Comment:
suggestion: use `get_dummy_request()` (`from wagtail.coreutils import get_dummy_request`)

I know we have a few tests still using `RequestFactory().get("/")` but we're trying to move to `get_dummy_request()`